DataManagementSystem/Utilities/CSHelpers.py

```diff
@@ -0,0 +1,235 @@
+"""
```

Comment:
This is a general question not target at Philippe specifically, but why do we need to have Helpers in the ConfigurationSystem.Client module AND then CSHelpers in the utilities of some system. Can't we centralize a bit all that ?